integration/tests/cook/test_basic.py

```diff
@@ -2655,3 +2655,26 @@ def get_debug_status_code():
             self.assertEqual(404, missing_resp.status_code, missing_resp.text)
         finally:
             util.kill_jobs(self.cook_url, [job_uuid], assert_response=False)
+
+
+    @unittest.skipUnless(util.supports_mesos_containerizer_images(), "Requires support for docker images in mesos containerizer")
+    def test_mesos_containerizer_image_support(self):
+        job_uuid, resp = util.submit_job(self.cook_url, container={'type': 'mesos', 'mesos': {'image': 'alpine'}})
+        try:
+            self.assertEqual(201, resp.status_code, resp.text)
+            job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
+            slave_host = job['instances'][0]['hostname']
+            slave_state = util.session.get(f'http://{slave_host}:5051/state.json').json()
```

Comment:
We have some code in `test_docker_port_mapping` for getting the agent host/port -- should we extract and use that here?

Reply:
Done